SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialVirtualWorkerTranslator.h

```diff
@@ -51,6 +51,12 @@ class SPATIALGDK_API SpatialVirtualWorkerTranslator
 
 	TWeakObjectPtr<UAbstractLBStrategy> LoadBalanceStrategy;
 
+	static void ApplyMappingFromSchema(TMap<VirtualWorkerId, WorkerInformation>& VirtualToPhysicalWorkerMapping,
```

Comment:
Why make this static and not just operate on the member variable directly?

Reply:
Used during new startup flow, so exposed it in this fashion.